### PR TITLE
Make sure to render default value of an object property

### PR DESCRIPTION
**Pull request by Baggz**

### CODE REVIEW

**Comment by Baggz** [comment]
[Default values not showing for primitive types](https://trello.com/c/FrOrD1PX/246-default-values-not-showing-for-primitive-types)

File: circle.yml [hunk 1]
```diff
@@ -2,20 +2,10 @@ general:
...
-    - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.6 10
-    - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.9 20
-    - sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.9 20
-    - case $CIRCLE_NODE_INDEX in 0) nvm install 0.10.48 ;; esac
-    - case $CIRCLE_NODE_INDEX in 0) nvm install 0.12.17 ;; esac
-    - case $CIRCLE_NODE_INDEX in 1) nvm install 4.6.1 ;; esac
-    - case $CIRCLE_NODE_INDEX in 2) nvm install 6.9.1 ;; esac
+machine:
+  node:
+    version: 5.7.0
```
Comment (XVincentX) [inline circle.yml]: Why this version? Could we stick with the minimal one used in Apiary (4.6.2)?

File: circle.yml [hunk 2]
```diff
@@ -2,20 +2,10 @@ general:
...
 
-test:
+dependencies:
```
Comment (XVincentX) [inline circle.yml]: This is happening by default. You can remove these lines.

File: fixtures/0074-object-of-primitive-type-with-default.md [hunk 3]
```diff
@@ -21,6 +21,22 @@
...
+                                            <div style="width:100%;height:auto;display:flex;flex-direction:row;flex-wrap:no-wrap;justify-content:flex-start;align-items:stretch;position:relative;margin-top:6px;">
+                                                <div style="height:auto;width:100%;display:flex;flex-direction:column;flex-wrap:no-wrap;justify-content:flex-start;align-items:flex-start;">
+                                                    <div style="width:100%;height:auto;display:flex;flex-direction:row;flex-wrap:no-wrap;justify-content:flex-start;align-items:stretch;position:relative;cursor:pointer;">
+                                                        <div style="height:auto;width:12px;display:flex;flex-direction:column;flex-wrap:no-wrap;justify-content:flex-start;align-items:flex-start;min-width:12px;max-width:12px;">
+                                                            <div style="width:12px;height:12px;background-repeat:no-repeat;background-position:left center;background-image:url([object Object]);background-size:7px 12px;"></div>
```
Comment (XVincentX) [inline fixtures/0074-object-of-primitive-type-with-default.md]: `url([object Object])` 🤔

File: circle.yml (hunk 1 above)
Comment (XVincentX) [inline circle.yml]: Also - please modify `package.json` to make sure version matches (4.6.2)

**Pull request merged**